CIScripts/Common/Invoke-NativeCommand.ps1

```diff
@@ -1,7 +1,7 @@
 function Invoke-NativeCommand {
     Param (
-        [Parameter(Mandatory = $true)] [ScriptBlock] $ScriptBlock,
-        [Parameter(Mandatory = $false)] [Bool] $AllowNonZero = $false
+        [Parameter(Mandatory = $false)] [Bool] $AllowNonZero = $false,
```

Comment:
We can use flag here instead of parameter (didn't know about it when I've written it in the first place):
```
function Foo
{
  param(
    [switch] $AllowNonZero 
  )

  if ($AllowNonZero) { ... }
}
```